waku/v2/protocol/filter/options.go

```diff
@@ -53,27 +54,49 @@ type (
 		wg             *sync.WaitGroup
 	}
 
-	FilterParameters struct {
+	FullNodeParameters struct {
 		Timeout        time.Duration
 		MaxSubscribers int
 		pm             *peermanager.PeerManager
+		limitR         rate.Limit
+		limitB         int
 	}
 
-	Option func(*FilterParameters)
+	FullNodeOption func(*FullNodeParameters)
 
-	FilterSubscribeOption func(*FilterSubscribeParameters) error
+	LightNodeParameters struct {
+		limitR rate.Limit
+		limitB int
+	}
+
+	LightNodeOption func(*LightNodeParameters)
+
+	SubscribeOption func(*SubscribeParameters) error
 )
 
-func WithTimeout(timeout time.Duration) Option {
-	return func(params *FilterParameters) {
+func WithLightNodeRateLimiter(r rate.Limit, b int) LightNodeOption {
+	return func(params *LightNodeParameters) {
+		params.limitR = r
+		params.limitB = b
+	}
+}
+
+func DefaultLightNodeOptions() []LightNodeOption {
+	return []LightNodeOption{
+		WithLightNodeRateLimiter(rate.Inf, 0),
```

Comment:
what would be these rate limits for status? can we set those as defaults

Reply:
In status-go it's set as 1 per second.

Reply:
ok, i was asking more from the point of the new per peer based rate limiter. Is that also set to 1 per sec?

Reply:
nvm, just noticed that you had already changed defaults.